examples/auth_example/auth_example_server/README.md

```diff
@@ -4,12 +4,65 @@ This is the starting point for your Serverpod server.
 
 To run your server, you first need to start Postgres and Redis. It's easiest to do with Docker.
 
-    docker compose up --build --detach
+```bash
+cd auth_example/auth_example_server
+docker compose up --build --detach
+```
+
+Use these credentials in order for the server to connect to the database:
+
+```yaml
+host: localhost
+user: postgres
+database: auth_example
+password: 9S8rYW7XeIA8bmGY9FBzOSLwQZtQEFNr
+```
+
+Create the local `password.yaml` file in the server `config` directory. `auth_example/auth_example_server/config/password.yaml` with the following content:
+
+```yaml
+development:
+  database: '9S8rYW7XeIA8bmGY9FBzOSLwQZtQEFNr'
+  redis: 'V7YogaG9K2rnIpS1odXIKrqsW8kkfddt'
+```
+
+Then apply the database migrations to the database by running the following command from the `auth_example_server` directory:
+
+```bash
+dart bin/main.dart --apply-migrations -r maintenance
+```
 
 Then you can start the Serverpod server.
 
-    dart bin/main.dart
+```bash
+dart bin/main.dart
+```
 
 When you are finished, you can shut down Serverpod with `Ctrl-C`, then stop Postgres and Redis.
 
-    docker compose stop
+```bash
+docker compose stop
+```
```

Comment:
Suggestion: Move this section to the end.

If the guide is followed step by step then this would disable the rest of the steps :)